Resources/Prototypes/_EinsteinEngines/Entities/Mobs/Customization/Markings/harpy.yml

```
@@ -0,0 +1,318 @@
+
+# Ears Markings
+- type: marking
+  id: HarpyWingDefaultHuescale
+  bodyPart: RArm
+  markingCategory: Arms
+  speciesRestriction: [Harpy]
+  coloring:
+    default:
+      type:
+        !type:CategoryColoring
+          category: Hair
```

yamllint (1.35.1)

[error] wrong indentation: expected 8 but found 10

(indentation)